src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/IntegerCopyNumberStateCollection.java

```diff
@@ -87,7 +94,11 @@ public boolean equals(Object o) {
 
         final IntegerCopyNumberStateCollection that = (IntegerCopyNumberStateCollection) o;
 
-        return that.alleles.equals(this.alleles) && that.copyNumberStates.equals(this.copyNumberStates);
+        return that.copyNumberStates.equals(this.copyNumberStates);
     }
 
+    @Override
+    public int hashCode() {
+        return copyNumberStates.stream().map(IntegerCopyNumberState::hashCode).reduce(1, (a, b) -> a * 31 + b);
```

Comment:
Since this class consumes the ref copy number (which can change from record to record in the future), the hash function and the equality test must take that into account. Also, please document this intended behavior accurately -- thanks!

Reply:
I usually just let IntelliJ automatically generate equals, hashCode, and toString, and then do some manual cleanup for style issues.

Reply:
Done. Since for now the reference copy number is hardcoded as a constant in the class, I will pass it to the hashCode and equals methods in the update for the next release